packages/babel-plugin-transform-classes/src/builtin-classes.js

```diff
@@ -0,0 +1,35 @@
+export default new Set([
+  "Array",
+  "ArrayBuffer",
+  "Boolean",
+  "DataView",
+  "Date",
+  "Error",
+  "EvalError",
+  "Float32Array",
+  "Float64Array",
+  "Function",
+  "In16Array",
+  "Int32Array",
+  "Int8Array",
+  "Map",
+  "Number",
+  "Object",
+  "Promise",
+  "Proxy",
```

Comment:
Is it worth it to include this here when `Proxy` can't get subclassed anyway?

Reply:
I don't think it makes any sense to maintain this list manually.

This is what I've used to [include them all](https://github.com/WebReflection/babel-plugin-transform-builtin-classes/blob/master/.babelrc), something you can produce on any browser via:

```js
JSON.stringify(
  Object.getOwnPropertyNames(window)
    .filter(name => /^[A-Z]/.test(name))
    .sort(),
  null,
  '  '
)
```
so that from time to time one can run that again and update the list.

Reply:
I initially did something like this, but then it would include things like `Math`, `Infinity`...
I will ask to include this list in the [Globals](https://github.com/sindresorhus/globals) package.

Reply:
Well, that filter might include typeof “function” too